save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityTableView.kt

```diff
@@ -0,0 +1,354 @@
+                            td {
+                                className = ClassName("align-middle text-nowrap")
+                                div {
+                                    className = ClassName("row m-0")
+                                    Link {
+                                        to = "/${FrontendRoutes.VULNERABILITY_SINGLE}/${cellContext.row.original.identifier}"
+                                        +cellContext.value
+                                    }
+                                }
+                                div {
+                                    className = ClassName("row m-0")
+                                    cellContext.row.original.tags.forEach { tag ->
+                                        div {
+                                            className = ClassName("rounded-pill px-1 mt-1 mr-1")
+                                            style = jso {
+                                                fontSize = 0.8.rem
+                                                color = Colors.VULN_PRIMARY.value.unsafeCast<ColorProperty>()
+                                                border = "0.01rem solid $color".unsafeCast<Border>()
+                                            }
+                                            +tag
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    column(id = "short_description", header = "Description".t(), { summary }) { cellContext ->
+                        Fragment.create {
+                            td {
+                                className = ClassName("align-middle")
+                                +cellContext.row.original.summary
+                            }
+                        }
+                    }
+                    column(id = "progress", header = "Criticality".t(), { severityNum }) { cellContext ->
+                        Fragment.create {
+                            td {
+                                className = ClassName("align-middle text-center")
+                                +"${cellContext.row.original.severityNum}"
+                            }
+                        }
+                    }
+                    column(id = "language", header = "Language".t(), { language }) { cellContext ->
+                        Fragment.create {
+                            td {
+                                className = ClassName("align-middle text-center")
+                                +"${cellContext.row.original.language}"
+                            }
+                        }
+                    }
+                    // don't show submitter on user profile page and on vulnerability collection page on owner tab (coz it's always current user)
+                    if (!isUserProfilePage && (!isVulnerabilityCollectionPage || !isOwnerTab)) {
```

Comment:
suggestion to move these checking to variables with meaningful name:
`val showCosvSubmitter`, `val showOrganization` and etc.

Reply:
Done